### ISSUE TITLE
[BUG][Python] python-fastapi 7.9.0 generates imports of UNKNOWN_BASE_TYPE

### ISSUE THREAD

**Issue by al3ad** [OpenAPITools/openapi-generator#20154]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

Generated files have an unknown_base_type import. 

##### openapi-generator version

7.9.0

##### OpenAPI declaration file content or url
The code for the yaml spec can be found here: https://github.com/bbc/tams/tree/main/api

##### Generation Details

This is a snippet of `tests/test_default_controller_impl`:
```
# coding: utf-8

from fastapi.testclient import TestClient


from openapi_server.models.collection_item import CollectionItem  # noqa: F401
#from openapi_server.models.one_of_video_flow_audio_flow_data_flow_multi_essence_flow import OneOfVideoFlowAudioFlowDataFlowMultiEssenceFlow  # noqa: F401
from openapi_server.models.unknownbasetype import UNKNOWN_BASE_TYPE  # noqa: F401


def test_flows_created_post(client: TestClient):
    """Test case for flows_created_post

    
    """
    unknown_base_type = openapi_server.UNKNOWN_BASE_TYPE()

```

##### Steps to reproduce

```
build-server-local-fastapi:
	docker run --user ${USER}:${GROUP} --rm \
                   -v ${PWD}:/local openapitools/openapi-generator-cli:v${GEN_VER} generate \
				   --additional-properties=apiNameSuffix=controller_impl \
                   -t /local/.templates/7.9.0/python_fastapi/.openapi-generator-server/ \
                   -i /local/openapi.yaml \
                   -g python-fastapi \
                   -o /local/python_fastapi/
```

**Comment by wing328** [comment]
thanks for reporting the issue.

can you please test it with the python client generator as well to see if there are similar import issues?

please also use v7.10.0 which is the latest stable version released a few days ago.

**Comment by wing328** [comment]
thanks for reporting the issue

should be fixed via #20165

please give it a try later with the latest master or snapshot version when you've time
